### PR TITLE
update pkgdown-netlify-preview action; bump to dev version

### PR DESCRIPTION
**Pull request by zkamvar**
This is partially a test to check that https://github.com/hubverse-org/hubverse-developer-actions/pull/5 works

### CODE REVIEW

**Comment by zkamvar** [comment]
The first push was with the non-dev version (because I had neglected to update it 🤦🏽‍♂️) and it built the docs as if they were the release. 

Luckily this means that I now have a proper test! MUAHAHAHA

**Comment by zkamvar** [comment]
YESSS this works:

deploy for reg version: https://66fc6823d9202cd95ddb0539--hubdevs-pr-previews.netlify.app
deploy for dev version: https://66fc698df50664dcf51b88f5--hubdevs-pr-previews.netlify.app

**Comment by annakrystalli** [comment]
Woohoo!! Awesomeness!

File: .github/workflows/pkgdown-netlify-preview.yaml [hunk 1]
```diff
@@ -56,13 +56,20 @@ jobs:
...
+        run: |
+          if [[ $(grep -c -E 'sion. ([0-9]*\.){3}' ${{ github.workspace }}/DESCRIPTION) == 1 ]]; then
+            echo 'dir=./docs/dev' >> $GITHUB_OUTPUT
+          else
+            echo 'dir=./docs' >> $GITHUB_OUTPUT
+          fi
       - name: Deploy PR preview to Netlify
         if: contains(env.isPush, 'false')
         id: netlify-deploy
         uses: nwtgck/actions-netlify@v2
```
Comment (annakrystalli) [inline .github/workflows/pkgdown-netlify-preview.yaml:70]:
```suggestion
        uses: nwtgck/actions-netlify@v3
```
Update to latest version?

**Review by annakrystalli — APPROVED** [review]
Looks good! Nice work 😄

**Pull request merged**